### DISCUSSION TITLE
Should Superset support MariaDB as a metadata store?

### DISCUSSION THREAD

**Discussion by sfirke** [apache/superset#26741] [General]
I saw [this answer to a Superset issue](https://github.com/apache/superset/issues/23483#issuecomment-1749708460) where using MariaDB as a metadata store was identified as the cause of not being able to upgrade Superset.  It links to an Apache Airflow issue exploring the same problem.  The Airflow project subsequently surveyed their community, determined that MariaDB was not widely used as a metadata store, and officially declared it unsupported as a backend (see the rest of that linked GitHub issue).

Someone posted in Slack today about having a problem getting Superset configured with MariaDB as the backend and it made me wonder what the project's official stance is.

For starters, are Superset builds tested with MariaDB?  If not I would think we should at least say in the docs that it is not officially supported.  Right now [this is the main mention I see in the docs](https://superset.apache.org/docs/intro/#backend-technology), implying that MariaDB _is_ supported.

**Comment by geido** [comment]
Thanks, do you have any references for that survey?

**Reply by sfirke** [reply]
I believe it's the [2022 survey results blog post ](https://airflow.apache.org/blog/airflow-survey-2022/), which was referenced in [this comment ](https://github.com/apache/airflow/issues/24247#issuecomment-1159816055)from Jarek Potiuk.  Four out of ~196 respondents reported using MariaDB, most used PostgreSQL or MySQL.
